channels/us.m3u

```diff
@@ -1910,3 +1922,11 @@ https://thegateway.app/YouToo/CueTones/playlist.m3u8
 https://thegateway.app/YouToo/YTamerica/playlist.m3u8
 #EXTINF:-1 tvg-id="KanalDisney.us" tvg-country="RU" tvg-language="Russian" tvg-logo="https://i.imgur.com/Q9KoVy9.png" group-title="Kids",Канал Disney (576p) [Not 24/7]
 http://188.40.68.167/russia/disney/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="US;LATAM" tvg-language="Spanish" tvg-logo="https://i.imgur.com/BGDFIFt.png" group-title="Sports",Contacto Deportivo
```

Comment:
Are you sure that the name of the channel is correct? Here's what I see.
![vlcsnap-2021-09-21-16h28m02s490](https://user-images.githubusercontent.com/7253922/134179556-c586d267-62ba-4f26-b919-eff00b7c0ff6.png)

And google says that Contacto Deportivo is the name of the program
![image](https://user-images.githubusercontent.com/7253922/134179629-6ce35d2b-7607-44b1-b3bc-6e361370de50.png)

Reply:
Hard to tell. It was discovered in Peru channel requests and is taken from here
https://moyobamba.com/contactodeportivo/

Reply:
Well, it's probably just a rebroadcast.